### ISSUE TITLE
Spec still talks about "payment app"

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/webpayments-payment-apps-api#151]
Can we please remove payment app from the spec. We've updated other specs to talk about payment handlers.    

Consider this a FPWD blocking issue.

**Comment by adrianhopebailie** [comment]
No, it's used intentionally. They are different things. A payment app is any app that handles payment requests forwarded to them by the UA.

A payment handler is a new feature of the web platform that handles PaymentRequestEvents.

Therefore a web app that employs a payment handler is a payment app. 

If you think that is not clear from the spec please can you suggest some edits?

**Comment by marcoscaceres** [comment]
Argh... edit would be to remove any mention of "payment app". It already causes us a bunch of needless fights about two months ago because it's mega confusing. 

I'd prefer it not be mentioned anywhere. If we want to about "apps", then we can talk about web applications that serve as payment handlers, but that's it.

**Comment by adrianhopebailie** [comment]
I don't agree but let's give the spec some airtime and take feedback from people that are seeing it for the first time.

I don't think this is a blocker for FPWD for that reason.

Request that you close and we gather feedback post-FPWD on how these new concepts are received.

**Comment by ianbjacobs** [comment]
I think the discussions that have happened have led us to being clearer in the spec. +1 to leaving it in the FPWD, and seeing if we get additional feedback.

Ian

**Comment by msporny** [comment]
@marcoscaceres wrote:
>  If we want to about "apps", then we can talk about web applications that serve as payment handlers, but that's it.

Agreed, we should minimize the amount of new terminology we require Web Developers to learn.
